index.html

```diff
@@ -17,25 +18,24 @@
 
 </header>
 <nav>
-    <a class=="menu" href="#">O Grubym</a>
-    <a class=="menu" href="#">Umiejętności Grubego</a>
-    <a class=="menu" href="#">Ulti Grubego</a>
-    <a class=="menu" href="#">Ostateczna Forma Grubego</a>
+    <a class="menu active"  href="index.html">O Grubym</a>
+    <a class="menu" href="news.html">Umiejętności Grubego</a>
```

Comment:
smiga to przekierowanie? nie musisz dac sciezki absolutnej z "./" na poczatku? czyli

`./news.html`

Reply:
Dziala i przeskakuje na tamta strone bez /, sprawdz